### PR TITLE
Add mobile:getContexts command to get full context list

### PR DESCRIPTION
**Pull request by imurchie**
Addresses https://github.com/appium/appium/issues/10757

### CODE REVIEW

**Comment by imurchie** [comment]
Documentation: https://github.com/appium/appium/pull/10814

File: lib/commands/context.js [hunk 1]
```diff
@@ -68,5 +68,33 @@ extensions.getNewRemoteDebugger = async function () {
...
+ */
+extensions.mobileGetContexts = async function () {
+  const curOpt = this.opts.fullContextList;
+  try {
+    // `appium-ios-driver#getContexts` returns the full list of contexts
+    // if this option is on
+    this.opts.fullContextList = true;
+    return await this.getContexts();
+  } finally {
+    // reseet the option so there are no side effects
```
Comment (jlipps) [inline lib/commands/context.js]: `reset`

Comment (mykola-mokhnach) [reply lib/commands/context.js]: Eagle eye

**Comment by imurchie** [comment]
Published in `appium-xcuitest-driver@2.85.0`.

**Pull request merged**